infrastructure/oss/code_actions.go

```diff
@@ -31,6 +32,11 @@ import (
 )
 
 func (i *ossIssue) AddCodeActions(learnService learn.Service, ep error_reporting.ErrorReporter, affectedFilePath string, issueRange snyk.Range) (actions []snyk.CodeAction) {
+	if reflect.DeepEqual(issueRange, snyk.Range{}) {
```

Comment:
I'm wondering if there is no better way than deepEqual comparison for this

Reply:
It's easiest here - we could also manually compare all contents, but I got lazy ;)